src/rest/server/pamAuth.go

```diff
@@ -139,6 +178,36 @@ func PAMAuthenAndAuthor(r *http.Request, rc *RequestContext) error {
 	return nil
 }
 
+func JwtAuthenAndAuthor(r *http.Request, rc *RequestContext) error {
+	c, err := r.Cookie("token")
```

Comment:
Not sure if passing token through cookie is the standard practice. IMO tokens should be passed through "Authorization" header using the "Bearer" schema.
Authorization: Bearer <TOKEN>

OpenAPI also supports this scheme. This also seems to be a standard -- https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#Authentication_schemes.

The authentication middleware can use "Authorization" header value to find what type of client it is.
Authorization: Basic XZY ==> basic authentication
Authorization: Bearer XYZ ==> token authentication

Reply:
Since OpenAPI recommends this I would agree, we should use the Authorization header, even though we are not implementing OAuth.

Reply:
It also seems that OpenAPI 2.0 that we are currently using does not support bearer tokens, only 3.0 does.

Reply:
See my latest commits